libs/abis/src/abis/CoWSwapEthFlow.json

```diff
@@ -0,0 +1,152 @@
+[
```

Comment:
Trimmed and moved `CoWSwapEthFlow.json` to our repo.

Old file size: 389kb
New file size: 3.4kb

Reply:
That's a big difference!